lib/jekyll-sitemap.rb

```diff
@@ -15,7 +15,13 @@ class JekyllSitemap < Jekyll::Generator
     def generate(site)
       @site = site
       @site.config["time"]         = Time.new
-      @site.config["html_files"]   = html_files.map(&:to_liquid)
+      begin
+          exts = @site.config["sitemap"]["extensions"]
```

Comment:
Looks like this name spacing (which I agree with) will be a breaking change? Can we make it backwards compatible?